### PR TITLE
Release notes for 1.27.01

### PR DESCRIPTION
**Pull request by dereknoblej**

### CODE REVIEW

**Comment by the2hill** [comment]
Going to need to update the index too I believe

File: api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-27-01-20200629.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.. version-v1.27.01-release-notes:
+
+v1.26.27, Oct 22nd, 2019
```
Comment (the2hill) [inline api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-27-01-20200629.rst:3]: Date and version need to be adjusted here

**Comment by rackernexus** [comment]
Build 157 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-a09d3aba8/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by rackernexus** [comment]
Build 158 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-654b8f51f/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by rackernexus** [comment]
Build 159 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-b0a8c5e89/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by rackernexus** [comment]
Build 160 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-2702452f9/docs/cloud-load-balancers/v1/). valid for 14 days

**Pull request merged**